securitycenter/api/tests/SecurityCenterTests.cs

```diff
@@ -0,0 +1,89 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Xunit;
+using Snippets;
+
+public class SecurityCenterTests
+{
+    [Fact]
+    public void CreateNotificationConfig_ShouldCreateConfig()
+    {
+        string configId = "csharp-create-config-test111";
+        Assert.NotNull(
+            CreateNotificationConfigSnippets.CreateNotificationConfig(
+                GetOrganizationId(), configId, GetProjectId(), GetTopic()));
+        DeleteNotificationConfigSnippets.DeleteNotificationConfig(GetOrganizationId(), configId);
+        Assert.True(true);
+    }
+
+    [Fact]
+    public void DeleteNotificationConfig_ShouldDeleteConfig()
+    {
+        string configId = "csharp-delete-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(
+            GetOrganizationId(), configId, GetProjectId(), GetTopic());
+        Assert.True(
+            DeleteNotificationConfigSnippets.DeleteNotificationConfig(GetOrganizationId(), configId));
+    }
+
+    [Fact]
+    public void ListNotificationConfig_ShouldReturnNonEmpty()
+    {
+        string configId = "csharp-list-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(
+            GetOrganizationId(), configId, GetProjectId(), GetTopic());
+        Assert.NotNull(
+            ListNotificationConfigSnippets.ListNotificationConfigs(GetOrganizationId()));
+        DeleteNotificationConfigSnippets.DeleteNotificationConfig(GetOrganizationId(), configId);
+    }
+
+    [Fact]
+    public void GetNotificationConfig_ShouldReturnConfig()
+    {
+        string configId = "csharp-get-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(
+            GetOrganizationId(), configId, GetProjectId(), GetTopic());
+        Assert.NotNull(
+            GetNotificationConfigSnippets.GetNotificationConfig(GetOrganizationId(), configId));
+        DeleteNotificationConfigSnippets.DeleteNotificationConfig(GetOrganizationId(), configId);
+    }
+
+    [Fact]
+    public void UpdateNotificationConfig_ShouldUpdateConfig()
+    {
+        string configId = "csharp-update-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(
+            GetOrganizationId(), configId, GetProjectId(), GetTopic());
+        Assert.NotNull(
+            UpdateNotificationConfigSnippets.UpdateNotificationConfig(
+                GetOrganizationId(), configId, GetProjectId(), GetTopic()));
+        DeleteNotificationConfigSnippets.DeleteNotificationConfig(GetOrganizationId(), configId);
+    }
+
+    private string GetOrganizationId() {
+        return "1081635000895";
+    }
+
+    private string GetProjectId() {
+        return "project-a-id";
+    }
+
+    private string GetTopic() {
+        return "notifications-sample-topic";
+    }

```

Comment:
I'm not sure we want all this info in the code base.
The project ID you can get from an Env Variable and see my comment on having a fixture. There you can create and organization and a topic on test setup and clean it up on tear down.